### ISSUE TITLE
Screen tarring and such

### ISSUE THREAD

**Issue by aomriches** [thommcgrath/Beacon#129]
![beacon](https://user-images.githubusercontent.com/46359530/50664704-fa862480-0f7b-11e9-9e23-426ef467f37b.JPG)

Due to that and Not being able to see the items I place in drops, I cannot customize any of the drops.

**Comment by aomriches** [comment]
some things are not showing up when I click on them, should I delete my current microsoft thing you sent and download the one you sent? I reinstalled beacon and still having the same problems.

**Comment by thommcgrath** [comment]
Yes. That's what I'm saying. This isn't Beacon's fault. The part of Windows responsible for drawing the window is bad in some way and needs to be repaired. The linked file _may_ fix it.

**Comment by aomriches** [comment]
I was wondering if my version of java could play a roll in the problem? (im not exactly the most educated on computers as you can probably tell.)

**Comment by aomriches** [comment]
I also did all of those things you said to do and nothing is changing.

**Comment by thommcgrath** [comment]
It’s definitely not Java. Beacon doesn’t use Java in the slightest.

You can try running Windows Update to make sure everything is up to date. I hope it is already, but some people choose to turn it off. Beyond that, you probably need to do a Windows repair install. It would preserve your data, but it’s not simple, and not always without side effects.

But Beacon simply asks the system to draw a control at the given coordinates. I believe it’s DirectDraw responsible for it. And DirectDraw isn’t doing what it’s told. I wish I could be more help, but there’s not much I can do.
